aries-site/src/utils/search.js

```diff
@@ -70,3 +74,14 @@ export const getCards = cardCategory => {
         : page.parent.name === cardCategory,
     );
 };
+
+/*
```

Comment:
curious about how you came up with this?

Reply:
about the getRelatedContent function?

Reply:
yes

Reply:
The idea is that: 
1. Each object in the structure array represents a page
2. Need a flexible place to store related content
- That page may have a variety of related content that might be a direct sibling, parent, child or distant. 
- For example, the Dashboards page's related content might consist of content related to alternate layouts, other page templates, components which compose a dashboard, may be even link to a page on charting or colors used in charting. 
- With that need for flexibility to fit a variety of cases, I propose we create a `relatedContent` property on the page object which is an array of page names (right now, page names are assumed to be unique and are being treated as a unique key -- if this content lived in a database or content management system, we'd define these relationships via a unique id -- eventually we need to transition these references to ids, rather than names).
3. We can't assume that content provided for a page is 100% complete. Therefore, we should plan for cases where that property is not present, is empty, or has 1-n related pages.

With those assumptions, the approach to the function says:
1. Find the first object in the `structure` array which has a name equal to the page in which I am interested
2. Try to get the relatedContent property for that object, if exists use it; if not, assign an empty array.
3. Loop through that array, finding each page contained in that array
4. Return the resulting array of page objects

Hopefully that explanation is sound and helpful. LMK.

Reply:
That helped alot thank you!